src/cache.h

```diff
@@ -5,10 +5,14 @@
 #include <utils/memutils.h>
 #include <utils/hsearch.h>
 
+#ifndef TS_FLEXIBLE_ARRAY_MEMBER
```

Comment:
I define our own macro so I can define it in the VS build file correctly regardless of whether the user has PostgreSQL source or just the binary to build against. That is, if they have the binary FLEXIBLE_ARRAY_MEMBER is not necessarily set to 1, which Visual Studio needs. Trying to set FLEXIBLE_ARRAY_MEMBER via command line throws redefinition errors, so this seems like the best solution.